rfcs/src/template.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: *Fill me with pretty name and a unique ident. E.g: New Feature (`new_feature`)*
+- Feature Request Issue: *Link to issue*
+- RFC PR: *Link to original PR*
+- Status: *One of the following: [Under Review | Unstable | Stable | Cancelled]*
+
+# Summary
+[summary]: #summary
+
+Short description of the feature, i.e.: the elevator pitch. What is this feature about?
+
+# User Impact
+[user-impact]: #user-impact
+
+Why are we doing this? How will this benefit the final user?
+
+ - If this is an API change, how will that impact current users?
+ - For deprecation or breaking changes, how will the transition look like?
+ - If this RFC is related to change in the architecture without major user impact, think about the long term 
+impact for user. I.e.: what future work will this enable.
+
+# User Experience
+[user-experience]: #user-experience
+
+What is the scope of this RFC? Which use cases do you have in mind? Explain how users will interact with it. Also 
+please include:
+
+- How would you teach this feature to users? What changes will be required to the user documentation?
+- If the RFC is related to architectural changes and there are no visible changes to UX, please state so.
+
+# Detailed Design
+[detailed-design]: #detailed-design
+
+This is the technical portion of the RFC. Please provide high level details of the implementation you have in mind:
+
+- What are the main components that will be modified? (E.g.: changes to kani-compiler, kani-driver, metadata, 
+  installation...)
+- How will they be modified? Any changes to how these components communicate?
+- Will this require any new dependency?
+- What are the corner cases you antecipate?
+
+# Rationale and alternatives
```

Comment:
This could just be part of "Detailed design".

Reply:
Hummm... That's an interesting point. I can see that in some cases that would make sense.

The reason I believe this is not inside "Detailed Design" because the "Rationale and alternatives" may go beyond the implementation details. In some cases, the alternative could be providing a completely different user experience to achieve the same purpose. Like [this rust RFC](https://rust-lang.github.io/rfcs/1861-extern-types.html).

Does it make sense?